.env.example

```
@@ -17,6 +17,7 @@ EMAIL_PORT=587
 EMAIL_SENDER=noreply@example.com
 EMAIL_PASSWORD=xxxx
 
+ENV=local
 GOOGLE_APPLICATION_CREDENTIALS=trl-storage.json
 GCS_BUCKET_NAME=trl-pdf-storage
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GCS_BUCKET_NAME key should go before the GOOGLE_APPLICATION_CREDENTIALS key

(UnorderedKey)